internal/sdkprovider/service/servicecomponent/service_component_data_source.go

```diff
@@ -65,6 +65,11 @@ func DatasourceServiceComponent() *schema.Resource {
 				ValidateFunc: validation.StringInSlice(service.KafkaAuthenticationMethodTypeChoices(), false),
 				Description:  userconfig.Desc("Kafka authentication method. This is a value specific to the 'kafka' service component").PossibleValuesString(service.KafkaAuthenticationMethodTypeChoices()...).Build(),
 			},
+			"kafka_ssl_ca": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "Kafka certificate used, 'project_ca' is the default, can be 'letsencrypt' when enabled",
+			},
```

Comment:
Hey. Could you please add a validation and the possible choices to the description? There is a `KafkaSslCaTypeChoices()` in the go client. 
This will put the values to the documentation and improve the `plan` validation. Thanks.

Reply:
Please have a look at `kafka_authentication_method`, it has all enum values too.